client/Windows/wf_client.c

```diff
@@ -272,6 +272,29 @@ static BOOL wf_pre_connect(freerdp* instance)
 	return TRUE;
 }
 
+static void wf_append_item_to_system_menu(HMENU hMenu, UINT fMask, UINT wID, const wchar_t* text,
+                                          void* dwItemData)
+{
+	// initial windows system item position where we will insert new menu item
+	// after default 5 items (restore, move, size, minimize, maximize)
+	// gets incremented each time this function is called
+	// or maybe could use GetMenuItemCount() to get initial item count ?
+	static UINT position = 6;
```

Comment:
not a big fan of hidden global variables, can't you store that somewhere in wfContext ?

Reply:
ok